### ISSUE TITLE
Animated images

### ISSUE THREAD

**Issue by stevesoltys** [icerockdev/moko-widgets#250]
From what I can see, there is no support for displaying animated images (GIFs).

It would be nice to have this, for loading animations and such. If you have any idea on how you'd like this implemented, I can probably help out.

**Comment by Alex009** [comment]
hi!
you can implement your own image loader, just like here - https://github.com/icerockdev/moko-widgets/tree/develop/widgets-image-network. with usage of any platform libs which support gifs in correct way.
But you should use dev version of moko-widgets to use this feature. latest public version 0.1.0-dev-19 not support it. But dev snapshot - support. 
[Here](https://github.com/icerockdev/moko-widgets/issues/238#issuecomment-640994735) described how to use dev snapshots

**Comment by stevesoltys** [comment]
Yeah, I think if we use [YYImage](https://github.com/ibireme/YYImage) it should be very easy to add basic GIF support.

I believe the only dependency is moko-resources supporting GIF images.

**Comment by Alex009** [comment]
@stevesoltys why you think that moko-resources should have it dependency? it just UI think about how draw image....`moko-resources` can store `files` and `images` and it can be animated gifs, but we should correct render it - it should implemented into widgets side with correct image loading library at platforms

**Comment by stevesoltys** [comment]
@Alex009 

Yeah, I think we agree. I'm only saying that `moko-resources` should support packaging GIF images inside the `images` resources folder (I believe it only works for JPG and PNG right now).

I agree that the rendering logic should be entirely in `moko-widgets`.
